src/System.Windows.Forms/src/System/Windows/Forms/Form.cs

```diff
@@ -1516,6 +1516,15 @@ public Form[] MdiChildren
             }
         }
 
+        /// <summary>
+        /// Gets or sets the anchoring for minimized MDI children.
+        /// </summary>
+        /// <value><see langword="true" /> to anchor minimized MDI children to the bottom left of the parent form (WinForms default); <see langword="false" /> to anchor to the top left of the parent form (Windows default).</value>
```

Comment:
```suggestion
        /// <value><see langword="true" /> to anchor minimized MDI children to the bottom left of the parent form; <see langword="false" /> to anchor to the top left of the parent form.</value>
```

The platform defaults need to be placed under `remarks`, e.g. https://docs.microsoft.com/dotnet/api/system.windows.forms.mdiclient.backgroundimage
Something like:
```xml
<remarks>
By default Windows Forms anchors MDI children to the bottom left of the parent form, whilst the Windows default is top left.
</remarks>
```